server/internal/config/storage.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+type StorageConfig struct {
+	StorageRoot string `group:"storage" default:"/oxidrive/storage/files"`
```

Comment:
```suggestion
	StoragePrefix string `group:"storage" default:"/files"`
```

I don't think it makes sense to put this much nesting in the storage root.
This needs to be valid for both FS and S3, and can be combined with a mount point (for FS)/bucket name (for S3) to get the final storage location.

Example with FS:
```
StoragePrefix: /files
StorageFsMountPoint: /oxidrive/storage

Final location: /oxidrive/storage/files
```


Example with S3 (path-style):
```
StoragePrefix: /files
StorageS3Bucket: oxidrive
StorageS3Host: s3.amazonaws.com

Final location: s3.amazonaws.com/oxidrive/files
```

Example with S3 (vhost-style):
```
StoragePrefix: /files
StorageS3Bucket: oxidrive
StorageS3Host: s3.amazonaws.com

Final location: oxidrive.s3.amazonaws.com/files
```

Reply:
This allows Oxidrive to reuse the base location (aka the bucket or mount point) for internal files, like internal backups or profile picture if we ever implement them

Reply:
👍